tests/unit/moduleapi/internalsecret.tcl

```diff
@@ -69,14 +69,14 @@ start_server {tags {"modules"}} {
 
     test {RM_Call of internal commands succeeds only for internal connections} {
         # Fail before authenticating as an internal connection.
-        assert_error {*unknown command*} {r internalauth.rm_call_withclient internalauth.internalcommand}
+        assert_error {*unknown command*} {r internalauth.internall_rm_call 0 internalauth.internalcommand}
```

Comment:
Maybe we should name those modes to make it more readable. Or alternatively, keep the 3 different commands and use the mode internally to call the shared code?

Reply:
Yeah I can keep the three commands and route internally